server/graphql/common/expenses.ts

```diff
@@ -531,6 +531,13 @@ export async function createExpense(
     throw new Unauthorized('Missing expense.collective.id');
   }
 
+  if (remoteUser.rolesByCollectiveId) {
+    const isMember = Object.keys(remoteUser.rolesByCollectiveId).includes(String(expenseData.collective.id));
```

Comment:
:ice_cream: Since `rolesByCollectiveId` is an object, it sounds simpler to check the key directly:
```suggestion
    const isMember = Boolean(remoteUser.rolesByCollectiveId[String(expenseData.collective.id)]);
```

or with lodash
```suggestion
    const isMember = has(remoteUser.rolesByCollectiveId, String(expenseData.collective.id));
```

Reply:
Thanks. Done. 👍🏽